prometheus-openstack-exporter

```diff
@@ -119,8 +119,19 @@ def get_clients():
 
     else:
         raise(ValueError("Invalid OS_IDENTITY_API_VERSION=%s" % ks_version))
+    try:
+        gnocchi_service = keystone.services.list(name="gnocchi")
+        log.debug("gnocchi service:" + str(gnocchi_service))
+        if len(gnocchi_service) > 0:
+            from gnocchiclient.v1 import client as gnocchi_client
+
+            gnocchi = gnocchi_client.Client(session=sess_admin)
+    except:
```

Comment:
Bare excepts are an anti-pattern, should be more specific about which exceptions we expect here. If we want to catch all "normal" exceptions, catching the base Exception class would be preferable to a bare except: